src/__tests__/Modal.spec.js

```diff
@@ -396,4 +403,27 @@ describe('Modal', () => {
     instance.destroy();
     wrapper.unmount();
   });
+
+  it('should render nested modals', () => {
+    isOpen = true;
+    isOpenNested = true;
+    const wrapper = mount(
+      <Modal isOpen={isOpen} toggle={toggle}>
+        <ModalBody>
+          <Modal isOpen={isOpenNested} toggle={toggleNested}>
+            Yo!
+          </Modal>
+        </ModalBody>
+      </Modal>
+    );
+
+    jasmine.clock().tick(300);
+    expect(wrapper.children().length).toBe(0);
+    expect(document.getElementsByClassName('modal-dialog').length).toBe(2);
+    expect(document.body.className).toBe('modal-open modal-open');
```

Comment:
will need to follow up with a proper fix for duplicate classes by either tracking active modals or using a prop as a flag for managing that body class.